### ISSUE TITLE
buffer: `foldmethod` is used as a buffer option

### ISSUE THREAD

**Issue by Milly** [vim-denops/deno-denops-std#269]
### Describe the bug

`'foldmethod'` is "local to window" option.
```vimhelp
						*'foldmethod'* *'fdm'*
'foldmethod' 'fdm'	string (default: "manual")
			local to window
			{not available when compiled without the |+folding|
			feature}
	The kind of folding used for the current window.  Possible values:
	|fold-manual|	manual	    Folds are created manually.
	|fold-indent|	indent	    Lines with equal indent form a fold.
	|fold-expr|	expr	    'foldexpr' gives the fold level of a line.
	|fold-marker|	marker	    Markers are used to specify folds.
	|fold-syntax|	syntax	    Syntax highlighting items specify folds.
	|fold-diff|	diff	    Fold text that is not changed.
```

But it is used as "local to buffer" option in the buffer module.

https://github.com/vim-denops/deno-denops-std/blob/016970ecf2ba51c04ae15110e22d17cca31765c4/buffer/buffer.ts#L60
https://github.com/vim-denops/deno-denops-std/blob/016970ecf2ba51c04ae15110e22d17cca31765c4/buffer/buffer.ts#L526

### Expected behavior

Use as "local to window" option, or remove it if unnecessary.

**Comment by Milly** [comment]
@lambdalisue 
What happens if we don't change the `'foldmethod'`? I couldn't figure it out from the tests.

**Comment by lambdalisue** [comment]
To be honest, I don’t remember at all. But I have a vague feeling that I might have been tweaking `foldmethod` by mistake instead of `undolevel` (if so, it make sense to keep undo history.)

**Comment by Milly** [comment]
Closes by #271
